Release/include/cpprest/http_headers.h

```diff
@@ -159,7 +159,7 @@ class http_headers
     {
         if (has(name))
         {
-            m_headers[name] =  m_headers[name].append(_XPLATSTR(", ") + utility::conversions::print_string(value));
+            m_headers[name] =  m_headers[name].append(_XPLATSTR(", ")).append(utility::conversions::print_string(value));
```

Comment:
Since `string::append()` modifies the string, we no longer need an assignment back into the map here. This can be fixed when merging.